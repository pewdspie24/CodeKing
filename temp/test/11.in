21
82
6567 9702 8625 8107 8814 1866 964 3779 1386 2430 8347 1209 4696 9605 3710 5440 8943 9976 3606 8942 9686 7664 6343 2647 9675 9742 4405 2481 765 2867 2769 7589 9037 7572 9033 2679 3567 8917 5145 5544 9642 7682 7084 7810 8432 6629 9345 5637 1620 3455 8885 8689 9133 3573 4038 2358 4646 6847 9829 2779 7814 3374 719 4780 4284 6905 6156 2486 5861 8671 883 1978 1245 1571 7163 2831 1390 7331 799 6930 4892 1471 
100
8839 9061 2828 9640 3780 6226 2940 7424 937 275 8318 7564 5435 8813 5894 2002 7367 1828 9968 9741 8750 5095 4610 9470 5783 3235 9391 1557 1178 5756 7570 371 9251 4671 9315 5183 8568 5427 4602 124 5390 3003 1468 125 347 2154 686 9397 4620 1904 5169 7264 4958 9056 8979 503 7855 5618 6284 6198 6163 2831 4441 9067 6828 9129 4086 4428 4097 2718 3697 7689 3579 2378 7950 274 7385 6845 3294 3442 7901 1518 2362 2475 382 473 7477 7499 2776 2317 2179 8734 1284 2031 7023 4068 9365 3612 172 9178 
74
4219 1834 8745 3969 9876 2216 7312 5768 1672 688 6323 2451 4267 4978 7663 1551 7184 8839 2494 3922 8827 249 2905 4053 5614 1789 6165 9349 9003 2548 7098 468 3424 5831 1030 7759 9498 762 8501 3079 721 1403 5084 3263 4772 5225 4924 795 4192 6264 9556 1139 5106 9065 8725 1797 6566 950 2771 2152 473 3748 6770 5126 3732 1936 4637 8933 9677 6068 1711 3013 3824 9098 
80
9913 2567 1461 8183 7983 1195 2960 3827 8330 7999 5785 4219 4510 5181 8064 889 3814 8334 2905 1994 7135 4884 4512 6971 4506 8796 5782 6425 917 6631 3323 9306 433 5807 8639 6167 7405 7083 4802 7361 4997 3714 9455 8686 2913 8598 6639 5606 9269 2165 1931 8232 2679 1294 7240 1431 8759 4028 1333 1084 7868 2284 1839 6775 1778 787 6300 1596 6161 1506 9716 2685 1454 3016 9688 266 3152 2517 833 4762 
58
1422 9107 8569 2555 6300 4418 7766 3350 8087 1863 4858 4569 9385 2371 3150 5020 6022 6367 586 8705 8143 2135 4885 1532 2345 9381 4698 2360 4317 5729 3009 1374 1660 1918 6135 9019 695 1195 9382 6638 9652 5740 7094 139 904 5763 6450 4469 9621 4554 951 8558 7536 7908 2282 6088 9594 9316 
92
3348 227 4610 7203 7155 1508 3186 1855 2289 2442 4691 6661 9270 9322 1312 5386 7778 7150 9693 2704 8109 4566 7310 2338 9342 4329 198 6369 3268 7555 163 5024 6545 4565 2566 4429 5009 1324 931 8818 3217 8452 9180 3601 1534 6263 8013 8107 5905 9340 3097 1719 5901 6358 575 5624 5799 4439 5148 1391 9712 8646 5759 3310 2727 2464 1084 5068 2950 5902 9885 6585 6311 3439 3012 1255 8944 5542 713 7770 632 8124 5983 6689 9716 7653 7672 3889 946 5483 2454 4708 
76
4332 1851 1678 1092 9098 2278 6096 9397 2088 7793 4949 8650 1812 8768 2126 5327 4553 2552 2158 5404 4754 1680 4426 8651 1122 7104 4276 7547 2130 9068 2592 1406 1302 880 2027 7377 9264 8239 988 3485 3001 6337 6234 4392 463 2282 4292 2247 2469 8435 2497 6689 8283 8398 3210 6036 7128 4263 5121 6326 4552 2433 3152 1522 5387 3567 341 8227 6438 5249 1636 9087 6540 3841 4629 2366 
77
7089 1719 7051 2402 6290 7072 8718 9317 9976 5293 694 8421 938 7207 3028 5232 461 5005 1722 6907 9860 1618 7639 7239 9711 1130 6732 6890 7519 8627 4246 8131 8959 6277 7372 1337 560 1342 5209 6488 2634 4615 1287 9113 5351 8618 3969 8165 1246 9007 1465 8242 3915 4205 2834 2743 2786 2351 2254 5108 3107 2681 4438 9271 4189 2396 6227 5517 3450 1037 7272 1578 4324 1597 9208 1332 7773 
81
1141 3590 5974 7980 745 9533 2004 9657 8510 7836 1758 5346 7360 5999 6425 766 1649 1281 7755 552 4967 1514 2276 7549 3937 9636 4490 3113 8341 9883 3177 3183 1608 2861 7164 1039 8475 5169 1397 3661 8396 8851 7875 6360 9401 9573 6710 8321 7991 9880 2024 8904 8128 3377 3692 6017 1917 3433 6962 9445 8083 1335 664 455 1355 2042 1481 4436 8778 6787 1031 4305 3779 8872 7192 9007 5305 2872 603 9264 4901 
57
972 4991 8283 9534 7501 3905 7644 2132 4599 2270 4678 3933 8518 6743 2693 5868 8857 8043 4956 5350 9696 3795 6036 8110 7880 8779 5805 9220 3222 8382 7312 6424 8969 7789 5875 8336 6924 901 6174 5603 3196 2539 2956 4617 6236 1736 2887 7344 8979 7110 685 6369 5125 8685 3122 9201 6450 
79
5712 5877 712 3831 8071 6400 5928 2100 7359 5942 1797 5024 2080 4795 717 715 5077 6080 9333 9922 9968 8496 9141 3271 4080 4354 1285 4633 4029 4431 5201 4972 9535 3693 7718 6054 6172 3312 2374 4753 6193 5616 1395 1117 3832 7663 8686 1215 1657 865 875 7129 4563 8281 4150 5368 7067 614 8891 257 5181 3307 6685 8492 5428 1344 6097 5921 8628 5973 9245 2439 9809 1680 8208 4529 1698 9165 9611 
75
9484 1867 2773 9933 2882 441 8401 3229 1904 5409 3110 7077 2628 4495 8576 3381 4761 595 4105 1212 2861 5698 7947 5538 6541 9881 3779 8651 3989 6251 186 5614 4676 9200 2388 7455 8110 9337 6695 6036 6512 4873 713 2618 6699 1317 2112 2599 2523 7842 1685 2676 4302 2784 9091 2251 7133 6886 6678 6333 8652 4358 9553 9677 2458 5055 5432 5380 2963 2448 3707 5426 1900 4783 199 
57
6596 2696 3166 8028 8296 3716 6813 7062 6304 3342 3881 3601 8069 4363 250 6830 4017 1337 4160 4169 3731 7975 2563 3076 9034 3319 7756 3819 2364 5083 5194 6110 4276 7581 8128 715 7760 8121 8251 1329 7675 9056 6619 2310 1071 4621 1569 788 5962 9365 2655 1913 937 9090 5809 4774 3624 
60
7897 8178 4559 8692 4857 9379 8552 3919 1604 5799 1267 9873 4810 6426 9988 1915 2280 3110 584 465 5183 7895 1253 9788 7263 8151 1873 1811 1229 3246 3262 3052 6089 680 5198 5099 4420 5406 4371 2114 2085 1485 238 5123 7908 1506 1754 3642 5358 7849 5111 6605 3062 4929 8798 3615 1240 9385 6336 6516 
100
6293 125 4804 1996 3149 2466 6842 5533 9970 9372 7396 2669 302 1580 5673 9478 7332 5062 1483 1103 4853 3249 3299 7706 5278 578 7052 7979 7934 895 881 9512 420 3652 1121 3676 9162 2523 9681 8197 8182 7917 2671 3565 9263 1054 8593 5408 3782 8766 3544 8019 3780 3984 9199 2188 2680 4347 6485 6463 2614 956 9396 3699 4454 1377 3119 1247 6346 7358 6932 9442 8660 2128 8131 8459 8380 5541 4858 7724 6356 481 8740 585 9154 7460 7119 2880 1208 7196 4680 4702 1420 747 9877 8942 6711 6598 4884 4030 
50
4863 1004 8475 9634 9047 8750 1762 8699 3610 4479 1940 7852 2793 5940 9393 4746 5778 5579 3600 6725 3015 6017 4736 3026 8895 7402 2438 6533 2191 7766 5921 9796 3615 8778 7951 664 3072 2743 6675 6487 5894 2201 6458 7832 4628 6159 5219 7391 1667 679 
54
9881 5257 1446 1607 4232 6293 8529 9334 3423 7876 8987 8593 694 6450 302 8557 4354 5537 3161 1416 2743 5284 8837 3327 1308 6250 5802 5369 5719 6553 2744 1219 6427 4830 9948 6386 8087 8432 1118 1754 3831 4759 2134 1545 9951 3663 9044 8913 4704 4348 592 9954 8738 9815 
89
5572 620 3088 4571 6900 7519 3146 7541 1205 8102 8927 1607 8337 4094 5814 4808 1455 9242 6071 5115 5991 9264 5966 8707 6837 4476 1159 3429 2642 4280 1075 2152 5664 9308 129 3943 1768 9631 2637 1927 1583 1030 1851 7140 3884 9788 379 4688 2679 5021 4199 2244 172 6005 5453 756 9213 6265 5963 8167 5477 5871 2386 3714 9539 2090 5011 4484 3949 4963 4545 5242 9133 1491 8173 9245 8570 4265 1307 260 3708 690 2367 4291 4562 3538 3241 770 4588 
51
9055 3104 7869 4031 5723 6404 7418 5633 9620 6199 4176 8858 6041 6426 9473 6535 5850 8315 8194 3371 1037 3041 7881 3342 1019 7567 6039 4914 9870 159 2001 9494 6892 2194 8210 6665 5612 2764 4348 4549 8441 5751 2452 4811 2831 8037 990 1595 6859 1544 1886 
77
5744 9574 5289 3165 7687 7395 4419 805 5466 3504 3451 8301 1812 3285 587 4454 9385 5245 6378 1803 7029 2429 3938 3381 8544 7657 2805 8811 4384 6453 470 1127 9851 2862 3331 3428 697 7969 8700 7508 9597 3701 1121 7691 911 6770 6575 8645 7003 8174 7335 6089 5494 9927 6097 9900 7002 8627 1242 8281 3367 806 2837 2321 8024 2230 3409 2590 7833 7284 8349 8667 532 3396 7334 3049 513 
85
1153 3577 553 6847 6934 354 9164 5680 387 7068 9668 1799 7346 1892 4106 8256 309 5558 4088 1956 9712 6846 5006 140 4224 8948 5667 1220 9202 7335 1648 2778 3743 7789 5187 2975 5576 8754 4712 7219 5515 421 7905 7672 593 6141 8537 1302 5881 3519 2863 9561 9595 2932 4247 9097 617 7378 6958 2851 9346 9046 7709 8810 3397 411 2746 4770 4973 9094 5664 6800 6063 8923 3694 5154 1422 631 8935 7824 4937 561 5344 7409 901 
