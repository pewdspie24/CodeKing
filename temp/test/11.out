Lema
Bi
Lema
Lema
Lema
Lema
Lema
Lema
Bi
Lema
Bi
Bi
Bi
Lema
Bi
Bi
Bi
Lema
Bi
Lema
Lema
